Feature

FaridAsgarov opened the pull request:
Made a new insertAfter method which inserts a new node after a position of a certain other node. Examples of it in main inserting after the 1st, middle and last positions
-Removed redundant if check (at previously line 108)

FaridAsgarov committed:
Update MyLinkedList.java

Added method insertAfter(nodeAfterWhichIsInserted, nodeToBeInserted) - the method takes in 2 parameters to determine what is the new Node which will be inserted and the position of the node after which it will be inserted

FaridAsgarov committed:
Update Main.java

Examples of the new insertAfter method which inserts a node after a certain node position

FaridAsgarov committed:
Removed redundant if check (at previously line 108)

asgarov1 commented on src/com/company/MyLinkedList.java:
```diff
@@ -99,14 +99,12 @@ public void insertAfter(LinkedListNode nodeAfterWhichIsInserted, LinkedListNode
-            }
+        if (temp.getNextNode() != null) {
+            nodeBeforeWhichIsInserted = temp.getNextNode();
+        }
```
you can remove the line 96, and replace line 102 - 104 with:

LinkedListNode nodeBeforeWhichIsInserted = temp.getNextNode();

Nothing bad is going to happen from rewriting null with null and it makes your code more compact

FaridAsgarov committed:
Update MyLinkedList.java

removed redundant code on previously line 96 and replaced previous lines from 102-104 with LinkedListNode nodeBeforeWhichIsInserted = temp.getNextNode(); on the new line 101
